app/main.py

```diff
@@ -1,6 +1,33 @@
+import requests
+import os
+
+from dotenv import load_dotenv
+
+
+load_dotenv()
+API_KEY = os.getenv("API_KEY")
+BASE_URL = "https://api.weatherapi.com/v1"
+FILTERING = "Paris"
```

Comment:
Consider making the `FILTERING` variable dynamic or configurable, rather than hardcoding 'Paris'. This will make the function more flexible and reusable for different cities.